pylearn2/datasets/hdf5.py

```diff
@@ -2,338 +2,467 @@
 Objects for datasets serialized in HDF5 format (.h5).
 """
 
-__author__ = "Steven Kearnes"
-__copyright__ = "Copyright 2014, Stanford University"
+__author__ = "Francesco Visin"
+__copyright__ = "Copyright 2015, Politecnico di Milano"
```

Comment:
IANAL, but I don't think by submitting code to Pylearn2 we waive our copyright. The original copyright holder still has rights to use their own code with a different license, for instance, and can prosecute for copyright violations.
By working for the University or studying there, however, we do, so most of the code is copyright "Université de Montréal".
Canonical/Ubuntu actually was under fire recently for asking that contributors to their open source repository transfer their intellectual property to Canonical.

In that particular case, I'm not sure what the status of Francesco's intellectual property actually is, but I'm pretty sure UdeM and Politectnico will not fight over it :) so it does not matter.